### PR TITLE
[legacy] vc: Bump 1.4.4

### PR DESCRIPTION
**Pull request by dennisklein**

### CODE REVIEW

**Review by rbx — APPROVED** [review]

**Comment by rbx** [comment]
actually, vc download failed. hiccup or something wrong? I'll rerun.

**Comment by dennisklein** [comment]
Hm, maybe url is wrong.

File: cmake/legacy.cmake [hunk 1]
```diff
@@ -242,10 +242,10 @@ ExternalProject_Add(hepmc
 )
 
 list(APPEND packages vc)
-set(vc_version "1.4.3")
+set(vc_version "1.4.4")
 ExternalProject_Add(vc
   URL https://github.com/VcDevel/Vc/releases/download/${vc_version}/Vc-${vc_version}.tar.gz
```
Comment (rbx) [inline cmake/legacy.cmake:247]:
```suggestion
  URL https://github.com/VcDevel/Vc/archive/refs/tags/${vc_version}.zip
```
🤷‍♂️

**Review by rbx — CHANGES_REQUESTED** [review]

File: cmake/legacy.cmake (hunk 1 above)
Comment (rbx) [reply cmake/legacy.cmake:247]: There's only the source code link for this version: https://github.com/VcDevel/Vc/releases/tag/1.4.4
Can we build from that?

Comment (rbx) [reply cmake/legacy.cmake:247]: Seems to be the same thing though.

Comment (rbx) [reply cmake/legacy.cmake:247]: I've tested this locally and it builds. I'll commit to see how it does on the CI. Can drop if anything is wrong.

**Review by rbx — APPROVED** [review]

**Pull request merged**